app/main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+
+    if len(split_command) != 3 or split_command[0] != "mv":
+        print("Invalid command")
+        return
+
+    file_to_move = split_command[1]
+    new_file_name = split_command[2]
+
+    if file_to_move == new_file_name:
+        print("Source and destination file names are the same.")
+        return
+
+    new_file_dir = os.path.dirname(new_file_name)
```

Comment:
Consider using `os.path.join` to construct the `new_file_dir` path. This will ensure that your application is cross-platform and handles path separators correctly, as mentioned in the checklist.